Installing-and-Configuring/Upgrading.md

```diff
@@ -1,163 +1,179 @@
 ---
 layout: default
 title: Upgrading Heads
-permalink: /Updating/
+permalink: /Updating
 nav_order: 99
 parent: Installing and configuring
 ---
 
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+
 ![Flashing Heads on an x230 at HOPE]({{ site.baseurl }}/images/Flashing_Heads_on_an_x230_at_HOPE.jpg)
 
-Upgrading Heads
+
+Acessing Heads Recovery shell
```

Comment:
why is the recovery shell the first item under Upgrading? The default method should be via the Heads menu; recovery doesn't give you any additional access/privilege, just the ability to flash manually

Reply:
@MrChromebox Better?

Reply:
> @MrChromebox Better?

yes, much :)